php-checks/src/main/java/org/sonar/php/checks/formatting/FunctionSpacingCheck.java

```diff
@@ -44,14 +44,14 @@ private void checkClosureSpacing(FormattingStandardCheck formattingCheck, AstNod
     Token lParenToken = node.getFirstChild(PHPPunctuator.LPARENTHESIS).getToken();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "lParenToken". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1854)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "lParenToken" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)